### ISSUE TITLE
Fix Pay and Play

### ISSUE THREAD

**Issue by SujithThirumalaisamy** [code100x/muzer#122]
- On error we should Ideally give them creds so that the next time they try to pay. We should not deduce them
- Duplicates can be added but show a warning to the user that its a duplicate song. Acceptance lead to re addition of the song

**Comment by Ameerjafar** [comment]
@SujithThirumalaisamy can you assign this issue to me

**Comment by SujithThirumalaisamy** [comment]
@Ameerjafar How is the progress? Can you do a draft PR?

**Comment by Ameerjafar** [comment]
@SujithThirumalaisamy working on it

**Comment by mohitejaikumar** [comment]
@SujithThirumalaisamy  by error , did you mean that user has did payment , but we were unable to add there song , or did you mean that user paid , we were able to add song as well but server crashed , something bad happened 

can you elaborate on this

**Comment by Ameerjafar** [comment]
@SujithThirumalaisamy sorry 😔😔 I have some issue in my laptop

**Comment by SujithThirumalaisamy** [comment]
> @SujithThirumalaisamy by error , did you mean that user has did payment , but we were unable to add there song , or did you mean that user paid , we were able to add song as well but server crashed , something bad happened
> 
> can you elaborate on this

Yeah. Payment has been done. But we are unable to play the song. So we failed to provide the value. so the next time we should not deduce them.
